Readme.mkd

```diff
@@ -72,6 +72,38 @@ $.ajax({
 });
 ```
 
+In case, if you need to find the rank of the object w.r.t to the ranked column, you could do this by accessing `{column_name}_rank` method available on the model objects, where `{column_name}` is your resource ranking column.
+
+The `row_order_rank` method will return the position of the duck object in the list w.r.t to the order defined by the row_order column
+
+``` ruby
+@duck.row_order_rank # 1, 2, 7, 17 .. so on
+```
+
+NOTE - The rank method would return 1 for the first duck in the list
+
+You could use the above interface to construct helper methods such as `insert_after` and `insert_before` on your model.
```

Comment:
I think the documentation from here down could be omitted as this is something you'd like (and I'd like) to add to the gem at a later date anyway. Better that we not encourage users to replicate it as that'll make upgrading more difficult in the future for them.

Reply:
Yes, definitely makes sense. Once this is merged will add the interfaces for adding `insert_after` and `insert_before`